### PR TITLE
Setup ci

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
Set up continous integration with travis and codecov.

If you agree to the PR, before to merge it, I'll add the CI badges to The README =)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 mock==2.0.0
 nose==1.3.7
 tox==2.6.0
+codecov
```
Comment (vdloo) [inline requirements.txt]: please pin this to a specific version, that way the build won't fail out of band if this dependency is upgraded and becomes incompatible for some reason.
```
pip freeze | grep codecov
codecov==2.0.9
```

**Comment by vdloo** [comment]
sounds good to me :+1:

**Comment by AdrieanKhisbe** [comment]
Hey @vdloo,
I pined to codecov version, and add CI badges.
Good to go 😃

**Comment by vdloo** [comment]
thanks!

**Pull request merged**